rnn_serving/rnn_serving.ipynb

```diff
@@ -0,0 +1,236 @@
+{
```

Comment:
<p>This should be stored in wasabi before it can be merged</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/mlrun/functions/pull/110/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>